makefile

```diff
@@ -1,33 +1,35 @@
 SHELL=/bin/bash
 
 # i.e. fedora28.yaml
-ALL_TEMPLATES=$(wildcard templates/*.yaml)
+ALL_META_TEMPLATES=$(wildcard templates/templates/*.yaml)
+ALL_TEMPLATES=$(wildcard templates/*.dist.yaml)
```

Comment:
We can have all built artifacts in `build/*` or so, then it's easier to distinguish it from source content.

Having tempaltes in `build/` (or `build/template/*`) would also make the readme nicer, as it would be: `kubectl apply -f build/windows.yaml`

Reply:
dist/ is the common name for such directory, sure I can change that. But then we should probably do a bigger change to the repo. Move the ansible file to the root directory and use the templates dir for the meta templates and dist dir for results.